stack_helpers.sh

```diff
@@ -58,6 +59,7 @@ EOF
     CMAKE_OPTS="-DCMAKE_INSTALL_LIBDIR=${CMAKE_INSTALL_LIBDIR}"
     CMAKE_OPTS+=" -DTMPL_FILE=$tmpl_file -DVERSION=$version"
     [[ -n "${py_version:-}" ]] && CMAKE_OPTS+=" -DPYTHON_VERSION=$py_version"
+    CMAKE_OPTS+="${module_substitutions}"
```

Comment:
Going with the suggestion from `build_mapl.sh`, as this will allow the user to be a bit lax on having to enforce a ` ` in the build script.
```suggestion
    CMAKE_OPTS+=" ${module_substitutions}"
```